src/main/java/Task.java

```diff
@@ -0,0 +1,26 @@
+public class Task {
```

Comment:
Would it help to make the `Task` class abstract? `Todo`, `Event` and `Deadline` extend `Task`, but you would never explicitly initialise a `Task` object.

Reply:
You're right, it would help. I've made it abstract.